perception/image_projection_based_fusion/test/test_geometry.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 TIER IV, inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "image_projection_based_fusion/utils/geometry.hpp"
+
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+
+TEST(objectToVertices, test_objectToVertices)
+{
+  // Test `boundingBoxToVertices()` and `cylinderToVertices()` simultaneously
+  // Test at Shape::BOUNDING_BOX
+  geometry_msgs::msg::Pose pose;
+  pose.position.x = 1.0;
+  pose.position.y = 2.0;
+  pose.position.z = 3.0;
+  const double q_scalar = std::sin(M_PI / 12);
+  pose.orientation.x = 0.0;
+  pose.orientation.y = 0.0;
+  pose.orientation.z = q_scalar;
+  pose.orientation.w = std::cos(M_PI / 12);
```

Comment:
How about this? Using a magic number `M_PI / 12` in two different lines may reduce the readability.
```suggestion
  const double angle = M_PI / 12;
  pose.orientation.x = 0.0;
  pose.orientation.y = 0.0;
  pose.orientation.z = std::sin(angle);
  pose.orientation.w = std::cos(angle);
```

Reply:
fixed in [4a4f8f6](https://github.com/autowarefoundation/autoware.universe/pull/7096/commits/4a4f8f68a2516389ee445f40e7d9291e9a52fe1d)